_data/people.yml

```diff
@@ -134,6 +134,11 @@
    email: carson.kohlbrenner@colorado.edu
    pic: carson.jpg
 
+ - name: Caleb Kumar
+   role: undergrad
+   email: caleb.kumar@colorado.edu
+   pic: caleb_kumar.jpg
```

Comment:
ok last thing: this should be the name of your image file! Eitehr change this or the image filename.